lib/commands/deploynodeapp.js

```diff
@@ -495,14 +804,16 @@ function handleUploadResult(err, req, fileName, itemDone) {
 
 // Create a target endpoint that references the Node.js script
 function createTarget(opts, request, done) {
-  var targetDoc = mustache.render(
```

Comment:
Did you mean to remove this?  Shouldn't `createTarget` generate a template based on the appropriate backend?